alfresco-health-processor-platform/src/test/java/eu/xenit/alfresco/healthprocessor/plugins/solr/SolrRequestExecutorTest.java

```diff
@@ -82,6 +84,37 @@ void checkIndexedNodes() throws IOException {
         assertEquals(SetUtil.set(), toDbIds(solrSearchResult.getDuplicate()));
     }
 
+    @Test
+    void checkIndexedNodesAndTransactions() throws IOException {
+        SearchEndpoint endpoint = new SearchEndpoint(URI.create("http://nowhere/solr/index/"));
+
+        List<Status> nodeRefs = new ArrayList<>();
+
+        nodeRefs.add(randomNodeRefStatus(10L, 1L));
+        nodeRefs.add(randomNodeRefStatus(11L, 2L));
+        nodeRefs.add(randomNodeRefStatus(100L, 5L));
+        nodeRefs.add(randomNodeRefStatus(1000L, LAST_INDEXED_TX + 1));
```

Comment:
Can you add an entry here for a node in transaction 2 that is not returned from the search response?

That node should also be considered as missing, even though a part of the transaction is indexed.

I know that no special checks are done with the transaction id right now, but adding that case will document & test the behavior that we expect.

Reply:
Ok will adjust

Reply:
The current/old implementation will also detect that case. I added another case in which an update to a document is missed.